src/main/java/com/dataloom/client/RetrofitFactory.java

```diff
@@ -13,10 +13,10 @@
 import retrofit2.Retrofit;
 
 public final class RetrofitFactory {
-    private static final String BASE_URL         = "https://api.thedataloom.com/datastore/ontology/";
+    private static final String BASE_URL         = "https://api.thedataloom.com/";
```

Comment:
won't the trailing slash be a problem? won't it conflict with all the `/datastore` paths in the interfaces?